docs/src/architecture/accounts.md

```diff
@@ -1,59 +1,61 @@
+    This account type will be used by most users. They can specify and change their code and use this account for connecting with a wallet. The Account ID will always start with `00`.
 
-### Account storage mode
-Account data - stored by the Miden Node - can be public or private. The third most significant bit of the ID specifies whether the account data is public `0` or private `1`.
+2. Regular account with immutable code
+    This account type will be used by most smart contracts. The code should not change after it is deployed, and no one should be able to update it. The Account ID will begin with the letter '01'.
 
-* Accounts with **public state**, where the actual state is stored on chain. These would be similar to how accounts work in public blockchains.
-* Accounts with **private state**, where only the hash of the account is stored on chain. The hash is defined as: \
-`hash([account ID, 0, 0, nonce], [vault root], [storage root], [code root])`
+3. Fungible asset faucet with immutable code
+    Assets need to be created by accounts in Miden. The Account ID will start with `10`. 
 
-### Example account ID
+4. Non-fungible asset faucet with immutable code
+    Assets need to be created by accounts in Miden. The Account ID will start with `11`.
 
-<p align="center">
-    <img src="../diagrams/architecture/account/Account_ID.png">
-</p>
+### Account Storage Modes
+Account data which is stored in a Miden Node can be either `public` or `private`. The third most significant bit ([MSb](https://en.wikipedia.org/wiki/Bit_numbering)) of the Account ID specifies whether the account data is public (`0`) or private (`1`).
+
+0. Accounts with **public state** store their actual state on chain, similar to how accounts work in public blockchains like Ethereum.
+1. Accounts with **private state** store only the hash of their account on chain. The hash is defined as: \
+`hash([account ID, 0, 0, nonce], [vault root], [storage root], [code root])`
 
 ## Storage
-User-defined data that can be stored in an account. Account storage is composed of two components. The first component is a simple sparse Merkle tree of depth 8 which is index addressable. This provides the user with 256 Word slots. Users that require additional storage can use the second component which is a `MerkleStore`. This will allow the user to store any Merkle structures they need. This is achieved by storing the root of the Merkle structure as a leaf in the simple sparse merkle tree. When `AccountStorage` is serialized it will check to see if any of the leafs in the simple sparse Merkle tree are Merkle roots of other Merkle structures. If any Merkle roots are found then the Merkle structures will be persisted in the `AccountStorage` `MerkleStore`.
+Storage refers to the user-defined data which can be stored in an account. Account storage is made up of two components. The first component is an index addressable basic sparse Merkle tree of depth 8. This gives the user 256 word slots.
```

Comment:
With `Storage` here I meant `AccountStorage`. So maybe to prevent confusion, we could say 

`Storage refers to 'AccountStorage' which is user-defined data ...`